components/AttachmentConversations/index.tsx

```diff
@@ -667,7 +676,7 @@ export const ImageConversations = ({
           onLongPress={handleLongPress}
           delayLongPress={200}
           onPress={event => {
-            handleOnPress(event, attachment0?.url);
+            handleOnPress(event, attachment0?.url, 0);
```

Comment:
why 0?

Reply:
index of image on which we direct carousel